data/seattle-temps.csv

```diff
@@ -1,8760 +1,8760 @@
-2010/04/24 07:00,46.9
-2010/04/24 08:00,48.7
-2010/04/24 09:00,50.5
-2010/04/24 10:00,52.3
-2010/04/24 11:00,54.0
-2010/04/24 12:00,55.6
-2010/04/24 13:00,56.7
-2010/04/24 14:00,57.7
-2010/04/24 15:00,57.8
-2010/04/24 16:00,57.7
-2010/04/24 17:00,57.0
-2010/04/24 18:00,55.7
-2010/04/24 19:00,53.6
-2010/04/24 20:00,51.8
-2010/04/24 21:00,50.7
-2010/04/24 22:00,49.7
-2010/04/24 23:00,48.8
-2010/04/25 00:00,48.0
-2010/04/25 01:00,47.2
-2010/04/25 02:00,46.6
-2010/04/25 03:00,46.1
-2010/04/25 04:00,45.6
-2010/04/25 05:00,45.2
-2010/04/25 06:00,45.6
-2010/04/25 07:00,47.1
-2010/04/25 08:00,48.8
-2010/04/25 09:00,50.7
-2010/04/25 10:00,52.4
-2010/04/25 11:00,54.2
-2010/04/25 12:00,55.7
-2010/04/25 13:00,56.9
-2010/04/25 14:00,57.8
-2010/04/25 15:00,58.0
-2010/04/25 16:00,57.9
-2010/04/25 17:00,57.2
-2010/04/25 18:00,55.9
-2010/04/25 19:00,53.8
-2010/04/25 20:00,52.0
-2010/04/25 21:00,50.8
-2010/04/25 22:00,49.8
-2010/04/25 23:00,48.9
-2010/04/26 00:00,48.1
-2010/04/26 01:00,47.4
-2010/04/26 02:00,46.7
-2010/04/26 03:00,46.2
-2010/04/26 04:00,45.7
-2010/04/26 05:00,45.4
-2010/04/26 06:00,45.8
-2010/04/26 07:00,47.3
-2010/04/26 08:00,49.0
-2010/04/26 09:00,50.9
-2010/04/26 10:00,52.6
-2010/04/26 11:00,54.3
-2010/04/26 12:00,55.8
-2010/04/26 13:00,57.0
-2010/04/26 14:00,57.9
-2010/04/26 15:00,58.1
-2010/04/26 16:00,58.0
-2010/04/26 17:00,57.3
-2010/04/26 18:00,56.0
-2010/04/26 19:00,53.9
-2010/04/26 20:00,52.1
-2010/04/26 21:00,50.9
-2010/04/26 22:00,49.9
-2010/04/26 23:00,49.0
-2010/04/27 00:00,48.2
-2010/04/27 01:00,47.4
-2010/04/27 02:00,46.8
-2010/04/27 03:00,46.3
-2010/04/27 04:00,45.9
-2010/04/27 05:00,45.5
-2010/04/27 06:00,46.0
-2010/04/27 07:00,47.5
-2010/04/27 08:00,49.2
-2010/04/27 09:00,51.0
-2010/04/27 10:00,52.8
-2010/04/27 11:00,54.6
-2010/04/27 12:00,56.0
-2010/04/27 13:00,57.2
-2010/04/27 14:00,58.1
-2010/04/27 15:00,58.3
-2010/04/27 16:00,58.1
-2010/04/27 17:00,57.4
-2010/04/27 18:00,56.2
-2010/04/27 19:00,54.0
-2010/04/27 20:00,52.2
-2010/04/27 21:00,51.0
-2010/04/27 22:00,50.0
-2010/04/27 23:00,49.1
-2010/04/28 00:00,48.3
-2010/04/28 01:00,47.5
-2010/04/28 02:00,46.9
-2010/04/28 03:00,46.4
-2010/04/28 04:00,46.0
-2010/04/28 05:00,45.7
-2010/04/28 06:00,46.2
-2010/04/28 07:00,47.6
-2010/04/28 08:00,49.3
-2010/04/28 09:00,51.1
-2010/04/28 10:00,52.9
-2010/04/28 11:00,54.6
-2010/04/28 12:00,56.1
-2010/04/28 13:00,57.3
-2010/04/28 14:00,58.1
-2010/04/28 15:00,58.3
-2010/04/28 16:00,58.2
-2010/04/28 17:00,57.5
-2010/04/28 18:00,56.2
-2010/04/28 19:00,54.1
-2010/04/28 20:00,52.2
-2010/04/28 21:00,51.0
-2010/04/28 22:00,50.1
-2010/04/28 23:00,49.2
-2010/04/29 00:00,48.4
-2010/04/29 01:00,47.7
-2010/04/29 02:00,47.0
-2010/04/29 03:00,46.5
-2010/04/29 04:00,46.1
-2010/04/29 05:00,45.8
-2010/04/29 06:00,46.3
-2010/04/29 07:00,47.8
-2010/04/29 08:00,49.5
-2010/04/29 09:00,51.3
-2010/04/29 10:00,53.1
-2010/04/29 11:00,54.8
-2010/04/29 12:00,56.3
-2010/04/29 13:00,57.4
-2010/04/29 14:00,58.2
-2010/04/29 15:00,58.5
-2010/04/29 16:00,58.
```

Comment:
I guess the completely correct format here would be `2010-12-31T23:00:00-08:00`, right? 

Unfortunately, the new format does not get parsed correctly in Safari. 

<img width="328" alt="Screen Shot 2019-12-17 at 15 14 18" src="https://user-images.githubusercontent.com/589034/71042212-f8ab6780-20df-11ea-8e8f-5034f6f398ec.png">

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Reply:
good catch! Datetimes are always trickier than I expect. I will try to fix thing up tomorrow.

Reply:
done!